examples/Basic usage.ipynb

```diff
@@ -185,8 +185,8 @@
    "outputs": [],
    "source": [
     "trial = 0\n",
-    "\n",
     "# Plot the mean image for one of the trials\n",
+    "plt.figure(figsize=(176.0 / 25, 156.0 / 25))\n",
```

Comment:
This is really overly complex. I guess you are trying to make the figure bigger without changing the aspect ratio? imshow will always have the right aspect ratio, so it is safe to just make a big square figure and then imshow will put the axes within that as best it can.

What you have now produces the same output as this:
```
plt.figure(figsize=(6.24, 6.24))
```

Reply:
I changed it to `figsize=(7, 7)`

Reply:
Yes I wanted to keep the aspect ratio. I did not know that this was automatically done!